recipes/looptools/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "looptools" %}
+{% set version = "2.16" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://feynarts.de/looptools/LoopTools-{{ version }}.tar.gz
+  sha256: 4113467575db3a14405d62d9e516b3b90410b73ea8d20c8eb8d70a30fc5cc9cb
+
+build:
+  number: 0
+
+outputs:
+  - name: {{ name }}-static
+
+    build:
+      skip: true  # [win]
+      run_exports:
+        - {{ pin_subpackage('looptools-static', max_pin='x.x') }}
```

Comment:
If it is static, it should need a run_exports.

Reply:
@xhochy Hm, I thought from https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements that I _was_ setting `run_exports` here. Is there more clear documentation on how this works?

Reply:
I was following `krb5` which is given as the example package in [CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md):

https://github.com/conda-forge/krb5-feedstock/blob/ee87321c98e795ffa99a9fc1abf1570272153e90/recipe/meta.yaml#L13-L15

Reply:
Hm, or should the `run_exports` be at the top level instead?

```diff
diff --git a/recipes/looptools/meta.yaml b/recipes/looptools/meta.yaml
index d9c5496de2..cd8b8240aa 100644
--- a/recipes/looptools/meta.yaml
+++ b/recipes/looptools/meta.yaml
@@ -11,14 +11,14 @@ source:
 
 build:
   number: 0
+  run_exports:
+    - {{ pin_subpackage('looptools-static', max_pin='x.x') }}
 
 outputs:
   - name: {{ name }}-static
 
     build:
       skip: true  # [win]
-      run_exports:
-        - {{ pin_subpackage('looptools-static', max_pin='x.x') }}
       ignore_run_exports_from:
         - {{ compiler('cxx') }}
       script:
```

Reply:
@xhochy Sorry, looking at the [`krb5` `run_exports`](https://github.com/conda-forge/krb5-feedstock/blob/ee87321c98e795ffa99a9fc1abf1570272153e90/recipe/meta.yaml#L13-L15) again I see that they are just for `krb5` not `krb5-static`.

> If it is static, it should need a run_exports.

Was this a typo, and you were indicating that they should not have `run_exports`?

Reply:
I'm going to assume "yes" given that the [`conda-build` docs mention](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements):

> Some build or host [Requirements section](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#requirements) will impose a runtime requirement. Most commonly this is true for shared libraries (e.g. libpng), which are required for linking at build time, and for resolving the link at run time.